bloom/block.go

```diff
@@ -0,0 +1,28 @@
+package bloom
+
+import "unsafe"
+
+// Word represents 32 bits words of bloom filter blocks.
+type Word uint32
+
+// Block represents bloom filter blocks which contain eight 32 bits words.
+type Block [8]Word
+
+// Bytes returns b as a byte slice.
+func (b *Block) Bytes() []byte {
+	return unsafe.Slice((*byte)(unsafe.Pointer(b)), BlockSize)
+}
+
+const (
+	// BlockSize is the size of bloom filter blocks in bytes.
+	BlockSize = 32
+
+	salt0 = 0x47b6137b
```

Comment:
Where did these values come from?

Reply:
From the spec https://github.com/apache/parquet-format/blob/master/BloomFilter.md#technical-approach